src/Components/Calculator/Calculator.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import './Calculator.css';
+import calculate from '../Logic/calculate';
+
+const Calculator = () => {
+  const [total, setTotal] = useState(null);
+  const [next, setNext] = useState(null);
+  const [operation, setOperation] = useState(null);
+  const handleClicks = (e) => {
+    const buttonName = e.target.textContent;
+    const output = calculate({ total, next, operation }, buttonName);
+    setNext(output.next);
+    setTotal(output.total);
+    setOperation(output.operation);
+  };
+  const Input = () => (
+    <div className="input-container">
+      <input type="text" value={next || total || '0'} className="input" />
+    </div>
+  );
```

Comment:
- [ ] Dear @vangelif, Because we provide a value for an input field, This will render a read-only field. according to react, if the field should be mutable we are expected to use `defaultValue`. Otherwise, we should set either `onChange` or `readOnly`. This asserts to react, what we are trying to accomplish here. 
### The error on the console
![image](https://github.com/vangelif/mm/assets/69077061/4fea3df6-8f62-4c81-8936-68cdc51852f2)